packages/edgeidentity/js/Identity.js

```diff
@@ -48,7 +48,18 @@ module.exports = {
    */
 
    getIdentities(): Promise<IdentityMap> {
-    return RCTAEPEdgeIdentity.getIdentities();
+    const getIdentitiesPromise = new Promise<IdentityMap>((resolve, reject) => {
+     
+      RCTAEPEdgeIdentity.getIdentities()
+      .then(identities => {
+        let identityMap = toIdentityMap(identities) 
+        resolve(identityMap);
+      })
+      .catch((error) => {
+        reject(error);
+      });      
+  });
+     return getIdentitiesPromise;
```

Comment:
can be simplified to return new Promise<IdentityMap>((resolve, reject) => { ... }